File: service/core/test/stores/TestData.scala
package stores

import io.lemonlabs.uri.Uri
import models.domain.{Collection, Institution, Object}

object TestData {
  val institution = Institution(uri = Uri.parse("http://example.com/institution"), name = "Test institution")
  val collection = Collection(uri = Uri.parse("http://example.com/collection"), name = "Test collection")
  val object_ = Object(uri = Uri.parse("http://example.com/object"), title = "Test object", titles = List("Test object"))
}

File: app/generic/test/integration/cypress/integration/TestData.ts
const objects = [...Array(10)].map((_, i) => ({
  subject: "Test subject " + i,
  title: "Test object " + i,
  uri: "http://example.com/object" + i
}));

export const TestData = {
  collection: {
    name: "Test collection",
    uri: "http://example.com/collection"
  },
  institution: {
    name: "Test institution",
    uri: "http://example.com/institution"
  },
  object: objects[0],
  objects
}
